### ISSUE TITLE
Schedule job dependency/chains

### ISSUE THREAD

**Issue by t0fik** [saltstack/salt#24719]
Situation: Processing of job1 on minion1 in normal conditions takes 40 to 70 minutes. After end of job1 on minion1 rest of processing has to be done on minion2 by job2.

It would be nice to have 'after' switch, which would start job2 after job1 ends or delays execution of job2 until job1 ends if job2 has own start times.

Example:
on minion1

``` yaml
job1:
  schedule.present:
    - function: cmd.run
    - job_args:
      - send_a_lot_of_stuff_to_minion2
    - when:
        - Monday 5:00pm
        - Tuesday 3:00pm
        - Wednesday 5:00pm
        - Thursday 3:00pm
        - Friday 5:00pm
```

on minion2

``` yaml
job2:
  schedule.present:
    - function: cmd.run
    - job_args:
      - do_something_with_stuff_from_minion1
    - when:
        - Monday 5:40pm
        - Tuesday 3:40pm
        - Wednesday 5:40pm
        - Thursday 3:40pm
        - Friday 5:40pm
    - after: minion1.job1
```

**Comment by jfindlay** [comment]
@t0fik, thanks for the feature request.

**Comment by kerncore** [comment]
Hi, 
You can use "Reactor System" or webhook's to init required job on required minion.
